tasks/get-plugins.yml

```diff
@@ -7,9 +9,17 @@
     owner: root
     group: root
 
+- set_fact:
+    plugin_url: "{{ cj_nexus_url }}/{{ cj_nexus_mirror }}"
+  when: cj_nexus
```

Comment:
cj_nexus_url already exist and its inherited from group_vars, functionally its working. its a code cosmetic issue